frontend/src/components/layouts/SettingsLayout.tsx

```diff
@@ -118,6 +118,9 @@ export default function SettingsLayout() {
             <MenuItem to="/settings/developer">Developer</MenuItem>
             <MenuItem to="/settings/debug-tools">Debug Tools</MenuItem>
           </nav>
+          <span className="font-medium slashed-zero text-muted-foreground text-sm">
```

Comment:
I noticed 2 minor things :
- on mobile the version placement doesn't look good
- when there's a scrollbar on desktop (e.g. the backup page) you can't see the version until you scroll down

Reply:
Shifted to Header content right

<table>
<tr>
<td><img src="https://github.com/user-attachments/assets/afda0fc7-e436-4afe-a879-259e918e86a8"/></td>
<td><img src="https://github.com/user-attachments/assets/d946beac-5316-4d07-8201-b20957641ff9"/></td>
</tr>
</table>